### PR TITLE
feat: Establish baseline repository

### PR DESCRIPTION
**Pull request by mmcgehee-ccc**

### CODE REVIEW

File: uuid.go [hunk 1]
```diff
@@ -0,0 +1,153 @@
+package cccutils
+
+import (
+	"github.com/go-playground/errors/v5"
+	"github.com/gofrs/uuid"
+)
+
+var NilUUID = UUID{}
```
Comment (bswaney-ccc) [inline uuid.go:8]: This variable can easily be mutated by any package that imports cccutils. Mutations in one package would go across all packages in the project. Maybe we should use a function or even just git rid of it.

```suggestion
func NilUUID() UUID {
	return UUID{}
}
```

Comment (mmcgehee-ccc) [reply uuid.go:8]: Isn't this the same thing that the standard uuid pkg does?

Comment (bswaney-ccc) [reply uuid.go:8]: I just looked into it and that is what they do, I take this one back.

**Review by bswaney-ccc — APPROVED** [review]
LGTM

**Pull request merged**